### PR TITLE
added AdminPages & filtering options

### PR DESCRIPTION
**Pull request by Parshana007**

### CODE REVIEW

File: backend/routes/stories.js [hunk 1]
```diff
@@ -66,4 +67,26 @@ router.post('/storysubmission', async (req, res) => {
     }
 })
 
+// Web scraping route
```
Comment (daiwik-swaminathan) [inline backend/routes/stories.js:70]: Why was this endpoint added? It seems to not be used/called anywhere...?

**Review by daiwik-swaminathan — CHANGES_REQUESTED** [review]
Looks good, but please see comment I left on the app.js file in the backend folder

File: frontend/src/pages/StoriesPage/StoriesPage.js [hunk 2]
```diff
@@ -52,10 +52,6 @@ function StoriesPage() {
 
     return (
         <div>
-            <StoryBanner
```
Comment (wbrandon25) [inline frontend/src/pages/StoriesPage/StoriesPage.js:55]: Restore the removed banner here.

**Comment by wbrandon25** [comment]
Great work with the story loading and sorting functionality. For the stories on this page, we want admins to be able to see important info about a story and decide whether they want to approve it or not - the categories won't be needed in this case. For now can you edit the stories list so that for every story we list out the the title, student major, and paragraph text with each story separated by a line break. Don't worry about css or formatting.

**Review by wbrandon25 — CHANGES_REQUESTED** [review]
The admin page and filtering buttons look great! Take a look at the comment Daiwik made about the webscraping endpoint and my comment on the banner above. Once those are addressed, we should be good to merge!

**Comment by Parshana007** [comment]
Sounds good I think the above change was from my last issue and this change I think I accidentally made since AdminPages and StoryPage were so similar. Thanks for the heads up I have fixed those two problems.

**Comment by daiwik-swaminathan** [comment]
I believe Brandon's request has been successfully addressed. However, the change I requested seems to not be addressed. Specifically, I do not see why the code in backend/routes/stories.js from lines 70-90 is there. If you could explain why this endpoint was added that would be great, otherwise it should be removed. Let me know if you have any questions/clarifications!

**Review by daiwik-swaminathan — APPROVED** [review]
Nice job!

**Pull request merged**